composer.json

```diff
@@ -5,8 +5,8 @@
     "homepage": "https://cakephp.org",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
-        "cakephp/console": "^4.0",
+        "php": "^8.1",
+        "cakephp/console": "5.x-dev",
```

Comment:
Should we have a 5.x branch for this tool?

Another oddity that I noticed about this file was that `type: rector-extension` in this file seems incorrect. This is more than just a rector extension. I also thought that the idea of rector extensions was on the way out.

Reply:
This doesn't need cake 5.

Reply:
@markstory I changed it to rector-extension in case that was required for rector to find the rules. I wasn't sure what was needed.

Reply:
ok. If it works with that type, I don't see why we need to change it.